pkgs/development/python-modules/openusd/default.nix

```diff
@@ -175,7 +175,7 @@ buildPythonPackage rec {
       for interchange between graphics applications.
     '';
     homepage = "https://openusd.org/";
-    license = lib.licenses.asl20;
+    license = lib.licenses.tost;
     maintainers = with lib.maintainers; [ shaddydc ];
```

Comment:
Given that you're familiar with the drv, you might want to add yourself as maintainer @gador

Reply:
Yes, maybe not a bad idea..